src/drivers/gitlab.js

```diff
@@ -536,6 +546,10 @@ class Gitlab {
 
     return await response.json();
   }
+
+  warn(message) {
+    console.error(message);
+  }
 }
```

Comment:
```suggestion
```

Reply:
Perhaps use `winston.warn` instead?

```suggestion

  warn(message) {
    winston.warn(message);
  }
}
```